src/test/fuzz/netbase_dns_lookup.cpp

```diff
@@ -63,9 +63,7 @@ FUZZ_TARGET(netbase_dns_lookup)
         assert(!resolved_service.IsInternal());
     }
     {
-        CSubNet resolved_subnet;
-        if (LookupSubNet(name, resolved_subnet)) {
-            assert(resolved_subnet.IsValid());
-        }
+        CSubNet resolved_subnet = LookupSubNet(name);
+        assert(resolved_subnet.IsValid() || !resolved_subnet.IsValid());
```

Comment:
`x || !x` will never be `false`. It suffices to change this to `(void)LookupSubNet(name);`.

Reply:
Done!